src/verifiers/documentStoreStatus/errors.ts

```diff
@@ -55,7 +70,10 @@ export const getErrorReason = (error: EthersError, address: string): Reason | nu
     (reason.toLowerCase() === "invalid address".toLowerCase() && error.code === errors.INVALID_ARGUMENT)
   ) {
     return contractAddressInvalid(address);
+  } else if (reason.toLowerCase() === "bad response".toLowerCase()) {
```

Comment:
There's the `errors.SERVER_ERROR` from Ethers, should I actually `&& error.code === errors.SERVER_ERROR`?